test/api/taskgroup_test.js

```diff
@@ -0,0 +1,257 @@
+suite('TaskGroup features', () => {
+  var debug       = require('debug')('test:api:taskGroup');
+  var assert      = require('assert');
+  var slugid      = require('slugid');
+  var _           = require('lodash');
+  var Promise     = require('promise');
+  var taskcluster = require('taskcluster-client');
+  var assume      = require('assume');
+  var helper      = require('./helper');
+
+  // Use the same task definition for everything
+  var taskDef = {
+    provisionerId:    'no-provisioner',
+    workerType:       'test-worker',
+    schedulerId:      'dummy-scheduler',
+    created:          taskcluster.fromNowJSON(),
+    deadline:         taskcluster.fromNowJSON('1 days'),
+    expires:          taskcluster.fromNowJSON('2 days'),
+    payload:          {},
+    metadata: {
+      name:           "Unit testing task",
+      description:    "Task created during unit tests",
+      owner:          'jonsafj@mozilla.com',
+      source:         'https://github.com/taskcluster/taskcluster-queue'
+    }
+  };
+
+  test("Create two tasks", async () => {
```

Comment:
is this testing anything related to the task group options? Just seems that it's making sure two tasks could be created.